src/components/screensavermanager.js

```diff
@@ -0,0 +1,134 @@
+define(["events", "playbackManager", "pluginManager"], function (events, playbackManager, pluginManager) {
+
+    function getMinIdleTime() {
+        // Returns the minimum amount of idle time required before the screen saver can be displayed
```

Comment:
pls add a notion of time units here, is it in milliseconds or what?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/592/commits/4038c9bcaef22a5adabc1d375c8dc79f4d54da1b